### PR TITLE
Update for offline + cleanup

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: src/app/github/store.ts [hunk 1]
```diff
@@ -13,15 +13,14 @@ import 'rxjs/add/operator/filter';
...
-  FirebaseListObservable, 
+import {
+  AngularFire,
   FirebaseObjectObservable,
-  FirebaseAuthState
 } from 'angularfire2';
 
 const EVENTS = '/github_webhook_events';
 const EVENTS_LEASE = '/github_webhook_events_lease';
+const LEASE_TIME = 1000;
```
Comment (mhevery) [inline src/app/github/store.ts]: Lease time should be something longer than `1000`  I know that is my fault, but we may as well fix it. Perhas every 30 seconds ?

File: src/app/+sync/sync.component.ts [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-import { FirebaseListObservable, FirebaseObjectObservable } from 'angularfire2';
-
 
 @Component({
   moduleId: module.id,
   selector: 'app-sync',
   templateUrl: 'sync.component.html',
-  styleUrls: ['sync.component.css']
+  styleUrls: ['sync.component.css'],
+  pipes: [AsyncPipe],
```
Comment (mhevery) [inline src/app/+sync/sync.component.ts]: It is a shame that we need to list the directives/pipes. I think @tbosch has a plan for this.

**Comment by mhevery** [comment]
LGTM

File: src/app/+sync/sync.component.ts (hunk 2 above)
Comment (vicb) [inline src/app/+sync/sync.component.ts]: Yep, it's in the Angular2 Core roadmap

File: src/app/github/store.ts (hunk 1 above)
Comment (vicb) [inline src/app/github/store.ts]: I'll switch back to 30s (I was indeed curious about the rationale for this change)

**Pull request merged**